### DISCUSSION TITLE
월드 제작시 월드 배경 에셋의 메테리얼이나 맵사이즈 가이드는 아이템 만들때랑 같나요?

### DISCUSSION THREAD

**Discussion by heodoly8672** [naverz/zepeto-studio-kor#1255] [5. Q&A]
안녕하세요,.
월드 제작시 월드 배경 에셋(예:건물, 바닥, 의자, 책상 등등)의 메테리얼이나 맵사이즈 가이드는 아이템 만들때랑 같나요.
아이템 만들때처럼 메테리얼도 제페토에서 만든 스크립트 (Standard.jr/ StandardTransparent_js)로 메테리얼을 사용해야하나요?
맵 사이즈도 최대 512로만 해야 될까요?

아니면 아이템 제작 방식과는 다르게 월드 제작에서는 좀더 자유롭게 메테리얼과 맵사이즈를 제작하려는 월드내에 맞게 결정하고, 사용할 수 있나요?
(유니티내에서 제공하는 기본 메테리얼을 사용해도 된다던가...맵사이즈를 512가 아닌 1024나 최대 2048로도 사용을 해도 되는지..)

**Comment by chlguswls32** [comment]
안녕하세요. heodoly8675님

제가 월드 검수받은기준으로, 유니티 스탠다스 shader와 커스텀(직접제작)한 쉐이더도 모두 사용가능했습니다.
맵사이즈도 아마 기준이있는것같지는 않네요. 모두 올라가는것같습니다. 제페토를 보통 모바일에서 플레이하니 최대 1024정도로 맞추는것이 최적화에 좋지않을까 생각합니다. (아무래도.. 2048은 512의 16배이다보니.. 그럴것같아요!) 

그럼 좋은하루되세요.

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, @chlguswls32 님께서 말씀해주신 것처럼 맵 사이즈 및 사용 가능한 메테리얼에 별도의 제약은 없습니다.
최대 용량 1기가 이내 및 최저 사양기기인 아이폰8/갤럭시 S8 에서 동작하도록 최적화 진행되면 자유롭게 사용이 가능합니다. 
관련 내용은 faq문서에서 도움이 될 것 같습니다.
https://studio.zepeto.me/guides/world-faqs
감사합니다.

**Reply by heodoly8672** [reply]
두분 모두 자세하고 친절한 답변 정말 ㄳ합니다.
